### ISSUE TITLE
[Android] “Authentication error. Try again!" message after removing the Android device from Windows VPN client

### ISSUE THREAD

**Issue by ValentinaPC** [mozilla-mobile/mozilla-vpn-client#189]
#### Affected versions:
* 2;

#### Affected Platforms:
* Android 10;

#### Prerequisites:
* Have the VPN app installed;
* Have a FxA account with active subscription for Mozilla VPN;
* Use VPN on 2 devices using the same account;

#### Steps to reproduce:
1. Open the VPN app on both Android and Windows and sign in using the account from prerequisites;
1. Go to “My devices” on the Windows VPN client;
1. Remove the Android VPN client;
1. Observe the VPN client from Android;

#### Expected result:
* User is signed out. A confirmation message is displayed.

#### Actual result:
* The user is signed out. The following error is displayed on a red banner: “Authentication error. Try again!"

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-78)

**Comment by jess-cook03** [comment]
@bakulf 
Based on conversation in review.... I would say MVP solution for this release since we are not able to tell that the user has deactivated the device (when the user then tries to access VPN on the device they deactivated).... we are not able to provide a contextual message at the moment and the general 'Auth' error message works fine for now -- keeping it as is.

I agree future iteration -- if we are able to (from the technical side) know and show a more contextual message to the user when they try to use VPN on a deactivated device? -- ie like a "This device is no longer active with your subscription. Manage or Change" ---- something like that in the future would be ideal if it's at all possible?

This is also an edge case as Chris pointed out.... how many users that actively/purposefully change the devices and then try to access VPN on the device they chose to deactivate will be confused by this? I assume not many, because most folks will have known they deactivated the device.

Still in the future to cover this edge case the contextual message would be a nice to have.

**Comment by ValentinaPC** [comment]
This is still happening. 
Tried removing the Android device from an iOS one => the user is singed out and the “Authentication error. Try again!" banner is displayed **ONLY after navigating to "My devices" page**.

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

We’ve updated the issue to “qa-major“ as, this issue could cause multiple after-issues like: app remaining  in “Connecting…“ state, features like, for example, contact support not working;

**Comment by data-sync-user** [comment]
➤ Beatriz Rizental Machado commented:

This issue is not Android specific. See: https://mozilla-hub.atlassian.net/browse/VPN-2388 ( https://mozilla-hub.atlassian.net/browse/VPN-2388|smart-link )

**Comment by data-sync-user** [comment]
➤ Sarah Bird commented:

Valentina Virlics can you try and repro. Should be resolved with 2.10.

**Comment by data-sync-user** [comment]
➤ Andrea Marchesini commented:

This is not about Android. A similar issue can be reproduced everywhere.
I think this is fixed already by the push-notification. @valentina can you please check?

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

Verified this as fixed using the last 2.11 builds on both Android and Windows 10.

While removing the Android device from Windows, user is signed out with success and not error is displayed. (Stage)
